ffupdater/src/main/java/de/marmaro/krt/ffupdater/app/impl/ThunderbirdBeta.kt

```diff
@@ -0,0 +1,61 @@
+package de.marmaro.krt.ffupdater.app.impl
+
+import android.content.Context
+import android.os.Build
+import androidx.annotation.Keep
+import androidx.annotation.MainThread
+import de.marmaro.krt.ffupdater.R
+import de.marmaro.krt.ffupdater.app.App
+import de.marmaro.krt.ffupdater.app.entity.DisplayCategory.FROM_MOZILLA
+import de.marmaro.krt.ffupdater.app.entity.LatestVersion
+import de.marmaro.krt.ffupdater.app.entity.Version
+import de.marmaro.krt.ffupdater.device.DeviceAbiExtractor
+import de.marmaro.krt.ffupdater.network.exceptions.NetworkException
+import de.marmaro.krt.ffupdater.network.github.GithubConsumer
+import de.marmaro.krt.ffupdater.network.website.MozillaArchiveConsumer
+import de.marmaro.krt.ffupdater.settings.DeviceSettingsHelper
+
+/**
+ * https://github.com/k9mail/k9mail.app
+ * https://github.com/thunderbird/thunderbird-android/releases
+ * https://api.github.com/repos/thunderbird/thunderbird-android/releases
+ */
+@Keep
+object ThunderbirdBeta : AppBase() {
+    override val app = App.THUNDERBIRD_BETA
+    override val packageName = "net.thunderbird.android.beta"
+    override val title = R.string.thunderbird_beta__title
+    override val description = R.string.thunderbird__description
+    override val installationWarning: Int? = null
```

Comment:
Probably missing an `installationWarning` string like Firefox beta and the other beta softwares have.  
`generic_app_warning__beta_version` looks like it would be used but it would require a wording/translate change as it uses the word "browser".


*Not a maintainer, I was going to try and add Thunderbird as my first contribution but I found this MR by a Thunderbird dev. Thanks for doing it first !*
*I can now only offer this unofficial review comment*

Reply:
Thank you, good point. I've generalized the beta warning used for Firefox so it could apply to basically any app added now and in the future. This might reduce the need for future translation changes.